packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -132,6 +133,7 @@ async function isAccountUpToDate(
     accountsWrapper.getWalletAddress(address),
     accountsWrapper.getDataEncryptionKey(address),
   ])
+  Logger.debug(`${TAG}/isAccountUpToDate`, `Found match for DEK ${currentDEK}`)
```

Comment:
We shouldn't be logging the DEK and also we don't know if we found a match. That's determined on the next line which checks equality

Reply:
Changed to `DEK associated with account ${currentDEK}`. Why shouldn't we log the DEK? From my understanding this is the public key pulled from the Accounts contract.

Reply:
That's true. I guess logging any kind of keys just makes me a bit nervous because these logs gets uploaded to Firebase. Though there's in theory nothing to be concerned about here.